spec/options_spec.rb

```diff
@@ -38,4 +37,78 @@ def calc
       }.to change { count }.by(1)
     end
   end
+
+  context 'disable caching options' do
+    # Reset options to default values after each test
+    after(:each) do
+      Kernel::silence_warnings { RedisMemo::DefaultOptions = RedisMemo::Options.new }
```

Comment:
I think it'd be clearer if we stub the entire `RedisMemo::DefaultOptions` object.

```ruby
stub_const("RedisMemo::DefaultOptions", new_options)
````